docs/pages/eas-update/github-actions.mdx

```diff
@@ -161,3 +161,41 @@ Your GitHub Action should be set up now. Whenever a developer creates a pull req
 > Some repositories or organizations might need to explicitly enable GitHub Workflows and allow third-party Actions.
 
 </Step>
+
+## Using Bun instead of Yarn
+
+If you're using Bun as your package manager instead of Yarn, you'll need to make some modifications to the provided workflow. Below are the updated steps for both publishing updates on push and previews on pull requests:
+
+Publish updates on push using Bun:
+
+<Step label="1">
+
+Replace the Node setup steps in update.yml:
+
+```- name: Setup Node
+  uses: actions/setup-node@v3
+  with:
+    node-version: 18.x
+    cache: yarn
+```
```

Comment:
```suggestion
```

I don't think we need to refer the same code block here. I've updated the context as to which step to modify by calling it out in the line above this snippet.